radionice/srednji/lekcije/030_kauzativ/030_kauzativ.tex

```diff
@@ -32,39 +39,85 @@
 	
 	Ova glagolska forma se koristi u dva konteksta:
 
-	1) kada se tjera nekoga nešto učiniti
-	2) kada se dopušta nešto nekome učiniti
+	1) kada se želi natjerati nekoga da nešto učini
+	
+	2) kada se želi dopustiti nekome da nešto učini
 	
 	\begin{reibun}
 		\rei{先生が学生に宿題をたくさんさせた。}{ Učitelj je natjerao učenike da pišu puno zadaće.}
-		\rei{先生が質問をたくさん聞かせてくれた。}{Učitelj je dopustio (nekome) da pita puno pitanja.}
+		\rei{先生が質問をたくさん聞かせてくれた。}{ Učitelj je dopustio (nekome) da pita puno pitanja.}
```

Comment:
Ovdje je točan odgovor vjerojatno bio maknuti vodeće razmake na ostalim retcima, a ne ovdje ga dodati

Reply:
Done